src/OpenTelemetry.Instrumentation.StackExchangeRedis/Implementation/RedisMetrics.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics.Metrics;
+using System.Reflection;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Instrumentation.StackExchangeRedis.Implementation;
+
+internal class RedisMetrics : IDisposable
+{
+    internal const string MetricRequestDurationName = "redis.client.request.duration";
```

Comment:
Is this expected to follow https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-metrics.md ?

Reply:
The naming of `redis.client.request.duration` follows the convention.

I'm sorry, I'm not sure I understand the question.

Reply:
the name from the semantic convention doc is "db.client.operation.duration" right? My question was, how did we pick the name "redis.client.request.duration" ?

Reply:
got it. sorry for my confusion, there are many implementations where the convention is not followed and this confused me, for example, cassandra (_db=cassandra_.client._operation=request_.duration).

I changed it to db.client.operation.duration.

Reply:
@cijothomas, can you check, please?

Reply:
Can you confirm if this is expected to follow https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-metrics.md, so I can review accordingly?

Reply:
Yes, this follows as expected. `db.client.operation.duration`. Could you review it, please?